src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -302,6 +303,12 @@ public FeedbackSessionAttributes updateFeedbackSession(FeedbackSessionAttributes
         // now, or else leave it as sent if so.
         if (oldSession.isSentOpenEmail()) {
             newUpdateOptions.withSentOpenEmail(newSession.isOpened());
+
+            // todo - help needed - not very sure what logic to use for resetting sentOpeningSoonEmail.
+            // also reset sentOpeningSoonEmail
+            newUpdateOptions.withSentOpeningSoonEmail(
```

Comment:
Hmm Im not sure why we need to reset `sentOpeningSoonEmail` if we alr reset `sentOpenEmail` above. I think the reason why we need to reset `sentOpenEmail` is to again remind owners to open the session (as its not opened), and this case does not apply to `sentOpeningSoonEmail` (as its not "opening soon" anymore)

Reply:
I see, my understanding was if a fb session had opened and the instructor wants to close it and reopen a few days later, the instructor should still be sent another opening soon email a day before the new opening date. 

Should we be only sending one opening soon email per feedback session even if the feedback session opens then closes before opening again?

Also should we be sending two opening soon emails if the following happens:
1. instructor is sent opening soon email 24h before session opens
2. instructor then changes the start time to a few days later, before that session opens
3. should an opening soon email be sent 24h before the new date?

Reply:
Hmm if we dont consider the cost, I would prefer being sent emails multiple times (at most 2 in this case), but many people might not like this. @damithc How do you think we should do?

Reply:
@pPris @daongochieu2810 Sorry I missed this earlier. Yes, we should send the email every time, as some instructors even reuse the same session multiple times. The extra cost of doing so is negligible as the increase in email count will be very small.

Reply:
@pPris great! then we can keep this here